src/components/Cluster/ClusterDetail/V4ClusterDetailTable.js

```diff
@@ -31,6 +31,47 @@ const WrapperDiv = styled.div`
   }
 `;
 
+const KubernetesURIWrapper = styled(FlexRowWithTwoBlocksOnEdges)`
+  flex-wrap: wrap;
+
+  .progress_button--container {
+    margin-right: 0;
+  }
+
+  & > div:nth-of-type(1) {
+    margin-right: 0;
+
+    & > * {
+      display: flex;
+    }
+  }
+
+  & > div:nth-of-type(2) {
+    margin-left: 0;
+    margin-right: 0;
+
+    ${mq(CSSBreakpoints.Large)} {
+      & > * {
+        margin-left: 0;
+      }
+    }
+  }
```

Comment:
Unfortunately I have to do things like this to override the high specificity in the not-so-extendable `FlexRowWithTwoBlocksOnEdges`.
I vote for refactoring this for the new cluster detail view screen

Reply:
Vote accepted!